src/screens/LoginScreen/index.js

```diff
@@ -39,7 +41,9 @@ const LoginScreen = () => {
                         Password:
                         <input className={"form-control m-1"} type={"password"}
                                onChange={(e) =>
-                                   setUserCredential({...userCredential, password: e.target.value})}/>
+                                   setUserCredential({...userCredential, password: e.target.value})}
+                               placeholder="password"
```

Comment:
Same here. But I do feel it makes sense to add placeholders in the register page ------ something like "minimum 4 characters" to give the username and password requirement ------ despite the fact we don't have any